services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from typing import List, Optional
+from django.db.models import DateTimeField
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: DateTimeField,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie=movie,
+                                cinema_hall=cinema_hall)
+
+
+def get_movies_sessions(
+        session_date: Optional[DateTimeField] = None
+) -> List[MovieSession]:
```

Comment:
The return type hint for `get_movies_sessions` is incorrect. This function returns a list of tuples containing `show_time` and `movie__title`, not `List[MovieSession]`. Consider changing the return type hint to `List[Tuple[datetime, str]]`.